ci/fix_wheels.py

```diff
@@ -4,7 +4,12 @@
 import zipfile
 
 try:
-    _, wheel_path, dest_dir = sys.argv
+    if len(sys.argv) != 3:
+        raise ValueError(
+            "User must pass the path to the wheel and the destination directory."
+        )
+    wheel_path = sys.argv[1]
+    dest_dir = sys.argv[2]
```

Comment:
Thanks @fabinca for your PR

Why does this need changing?

The rest seems fine

Reply:
Without this change it didn't pass the pre-commit checks: 
`ci/fix_wheels.py:11:4: W0632: Possible unbalanced tuple unpacking with sequence: left side has 3 label(s), right side has 2 value(s) (unbalanced-tuple-unpacking)`

Reply:
thanks - might be a bug in pylint, I've reported here https://github.com/PyCQA/pylint/issues/7710, let's see what they say

Reply:
seems like they'll keep the warning - let's just have an in-line `pylint: disable=` comment then?